### ISSUE TITLE
Develop overview of platform governance & underlying values

### ISSUE THREAD

**Issue by joedeville** [COPIM/open-book-collective#8]
As a librarian or institutional decision-maker visiting an open access initiative’s profile page ...

... I want a clear understanding about how the platform itself is administered and controlled.

E.g.

    What are the ‘entry requirements/standards’ for publishers/institutions hosted on the platform?
    Who makes those decisions?
    How does my own institution participate in that processes?
    What obligations are incurred with my participation

... so that I can be confident in my understanding of the organisation and be able to accurately justify membership in spending audits.

**Comment by judith-fathallah** [comment]
What are the ‘entry requirements/standards’ for publishers/institutions hosted on the platform? 

Publishers: a peer-reviewed OA publishing scheme, usually comprising at least 50% of the publishers outcome, though the board will consider applications from publishers with a smaller percentage of OA output if the publisher intends to develop this. Agreement with the core values of the platform.

Any library that agrees to subscription fee

Who makes those decisions? The platform governing body, on which see About page edits on the Wiki: https://github.com/COPIM/open-book-collective/wiki/About-Open-Book-Collective

How does my own institution participate in that processes? Again see https://github.com/COPIM/open-book-collective/wiki/About-Open-Book-Collective

What obligations are incurred with my participation

Librarians - to keep up fees
publishers to provide catalogue info, metadata according to standard format, access, info on upcoming books

**Comment by MartinPaulEve** [comment]
> What are the ‘entry requirements/standards’ for publishers/institutions hosted on the platform?
> 
> Publishers: a peer-reviewed OA publishing scheme, usually comprising at least 50% of the publishers outcome, though the board will consider applications from publishers with a smaller percentage of OA output if the publisher intends to develop this. Agreement with the core values of the platform.

Might this not prohibit presses that want to start making a transition but who are not at the threshold yet? I think we want to incentivize good behaviour and this might push legacy presses who are beginning a shift to the margins...

**Comment by judith-fathallah** [comment]
Well, that's why I put that we would consider publishers with a smaller % who want to develop it, but if you'd like to reword it to make that stronger please feel free!

**Comment by judith-fathallah** [comment]
Maybe:

Publishers: a sustainable, peer-reviewed OA publishing scheme and a commitment to the core values of the platform

But is that too vague now? Like how do they demonstrate said commitment/sustainability?

**Comment by PenierI** [comment]
Can we say publishers aiming to put out a minimum xxx OA books a year/each year in 2 coming years. We could also add a maximum limit to bar excess to commercial publishers and remain 'true' to 'scaling small' idea.

**Comment by judith-fathallah** [comment]
Sounds sensible. What should the minimum be? Some small publishers might only do a few titles.

**Comment by PenierI** [comment]
The biggest publishers from ScholarLed will publish 30+ books a year. So 40 could be the upper limit. I don't know how many books Mattering Press puts out, but Joe could advise on the lower limit.

**Comment by judith-fathallah** [comment]
Yeah I was just looking up in the report where you said that was the max but didn't see a min, we will ask Joe for that data when he gets back. I think this would be a good guideline, but maybe we should hedge it by phrasing it like 'we normally expect an output of between x-y, but the board will consider other schemas if the publisher can demonstrate appropriate commitment to values' or something

**Comment by judith-fathallah** [comment]
To not put people off applying

**Comment by PenierI** [comment]
Yes but I would avoid sounding dogmatic and putting eligibility criteria that  are difficult to measure.

**Comment by judith-fathallah** [comment]
Okay how about putting it like: "OBC aims to support small and medium publishers with a diverse range of revenue models. Most of our member publishers produce between x and y OA books per year. However, we do consider applications from publishers producing more or fewer OA books than this, provided they are operating in accordance with [our values](https://github.com/COPIM/open-book-collective/wiki/About-Open-Book-Collective). All applications are considered on a case by case basis by [our governing board](https://github.com/COPIM/open-book-collective/wiki/About-Open-Book-Collective)"

**Comment by judith-fathallah** [comment]
Me and Eileen are working on a shared document on this, off github. Will update the github with it when we're done

**Comment by judith-fathallah** [comment]
Okay the governance structures are going to be discussed in workshops in collab with Wp4, in May and July, so hopefully by the beginning of July we will have answers for this. We are compiling a doc of questions and issues, and the stakeholders that need to be balanced against each other.
